src/player.cpp

```diff
@@ -1971,6 +1971,11 @@ std::string player::save_info() const
 
 void player::memorial( std::ostream &memorial_file, std::string epitaph )
 {
+#ifdef _WIN32
```

Comment:
Please move this build logic to filesystem.cpp with a wrapper method, I don't want to have conditional compilation creeping into random game files.